src/components/orders/AmountsDisplay/index.tsx

```diff
@@ -40,7 +43,11 @@ function Row(props: RowProps): JSX.Element {
       <RowContents>
         <span>{formattedAmount}</span>
         {/* <UsdAmount>(~${usdAmount})</UsdAmount> */}
-        <TokenDisplay erc20={erc20} network={network} />
+        <RowWithCopyButton
+          textToCopy={erc20.address}
+          visible={!isNativeToken(erc20.address)}
+          contentsToDisplay={<TokenDisplay erc20={erc20} network={network} />}
+        />
```

Comment:
It would make more sense to render a different component if the copy button shouldn't be displayed, don't you think?

I suggest to put the TokenDisplay component in a const and use it like:

```ts
{isNative ? tokenDisplay : <RowWithCopyButton ... contentsToDisplay={tokenDisplay} />}
```